modules/libcom/RTEMS/posix/rtems_init.c

```diff
@@ -611,20 +613,10 @@ static void nfsMountCallFunc(const iocshArgBuf *args)
         }
         *cp = '/';
     }
-    nfsMount(args[0].sval, args[1].sval, args[2].sval);
+    iocshSetError(nfsMount(args[0].sval, args[1].sval, args[2].sval));
 }
 #endif
 
-
-void zoneset(const char *zone)
```

Comment:
Please reverse this change and keep `zoneset()` as a separate function.

There is method to this seemingly unnecessary verbosity.  RTEMS has alternate shells which directly lookup and execute functions by symbol names.

Changing the return type to `int` is OK.

Reply:
Done